probe-rs/targets/SF32LB58_Series.yaml

```diff
@@ -22,8 +30,8 @@ variants:
   - !Nvm
     name: Flash4
     range:
-      start: 0x68000000
-      end: 0xA0000000
+      start: 0x18000000
+      end: 0x1C000000
```

Comment:
What does this have to do with the AP value?

Reply:
> What does this have to do with the AP value?

Yes, it isn't related, but it is also related to debug to sf32lb58. I can change the commit message to reflect it, after @HalfSweet tested